encuentro-21/ejercitacion_na/edades.js

```diff
@@ -0,0 +1,25 @@
+let cantidadMayores = 0
+let cantinadMenores = 0
+let edad = 1
+
+function esMayor(edad) {
+  if (edad >= 18) {
+    return true
+  }
+  return false
+}
```

Comment:
Esto esta bien pero cuando te encuentre con este patron:
```
if (expresion) {
  return true
} else {
  return false
}
```

podes hacer simplemente `return expresion`

```suggestion
function esMayor(edad) {
  return edad >= 18
```